adapter_pipelines/submit_stub/submit.wdl

```diff
@@ -0,0 +1,27 @@
+task submit_stub {
+  command <<<
+    echo "This is a stub submission task for testing purposes only."
+  >>>
+  runtime {
+    docker: "python:2.7"
```

Comment:
If this stub `submit.wdl` doesn't do anything indeed, I would recommend using a smaller docker image here for minimizing the cost. [alpine](https://hub.docker.com/_/alpine/) could be a good candidate here.

Reply:
Tried alpine and also bash image and they're not working. Fortunately, this job will call cache every time since inputs never change, so image will only need to be pulled once.

Reply:
Glad to know that, thanks!